common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4376,6 +4380,40 @@ object CelebornConf extends Logging {
       .longConf
       .createOptional
 
+  val WORKER_OPEN_HDFS_OUTPUT_STREAM_MAX: ConfigEntry[Int] =
+    buildConf("celeborn.worker.openHdfsOutputStream.max")
+      .categories("worker")
+      .doc("If the number of opened hdfs output streams on a worker exceeds this configuration value, " +
+        "the worker will be marked as high-load in the heartbeat report, " +
+        "and the master will not include that node in the response of RequestSlots.")
+      .version("0.7.0")
+      .intConf
+      .createWithDefault(10000)
```

Comment:
The default value of 10000 is too large; the default size of a Java thread stack is 1MB, which means that this will consume approximately 10GB of memory.

Reply:
Maybe the default value can be set to 1000.